.github/workflows/ci.yaml

```diff
@@ -5,6 +5,38 @@ on:
     pull_request:
 
 jobs:
+    "typecheck":
```

Comment:
The double quotes are unnecessary but I've put them to follow the existing formatting. Should I open a PR to remove all the unnecessary double quotes?